cmd/describe/machinepool/cmd_test.go

```diff
@@ -48,8 +49,9 @@ Subnet:
 Version:                               4.12.24
 Autorepair:                            No
 Tuning configs:                        
-Message:                               
 Additional security group IDs:         
+Node drain grace period:               
```

Comment:
@philipwu08 We should extend this test to ensure that the `Node drain grace period` is correctly printed when it is set 👍